pkg/subctl/cmd/gather/cni.go

```diff
@@ -0,0 +1,161 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gather
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/resource"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	typeIPTables = "iptables"
+	typeOvn      = "ovn"
+	typeUnknown  = "unknown"
+	libreswan    = "libreswan"
+)
+
+var ipCmds = map[string]string{
```

Comment:
We would need ip-routes from all the route-agent pods, but the other two commands (ip-rules and ip-rules-table150) are generally only present on active Gateway node. Anyways, there is no harm in collecting them - might also help us to know if there are any cleanup issues during gateway migration.

Reply:
yeah, let's keep them just in case :)

Reply:
Yeah, on rest it will capture no output and we will not log it. It is easier and faster to just try grab on all then figure out which is which.

Reply:
Please let's keep `ip rule show table 150`   it will be useful for OVN on gateways.